airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/steps/gradle.py

```diff
@@ -97,12 +87,17 @@ async def _run(self) -> StepResult:
             "rsync",  # required for gradle cache synchronization.
         ]
 
-        # Define a gradle container which will be cached and re-used for all tasks.
-        # We should do our best to cram any generic & expensive layers in here.
-        gradle_container = (
+        # Common base container.
+        gradle_container_base = (
             self.dagger_client.container()
             # Use a linux+jdk base image with long-term support, such as amazoncorretto.
             .from_(AMAZONCORRETTO_IMAGE)
+            # Mount the persistent cache volume, but not to $GRADLE_HOME, because gradle doesn't expect concurrent modifications.
+            .with_mounted_cache("/root/gradle-cache", self.persistent_cache_volume, sharing=CacheSharingMode.LOCKED)
```

Comment:
Can we rename `/root/grade-cache` to `/root/gradle-dependency-cache`?
 And `persistent_cache_volume` to `dependency_cache_volume`?

The "persistency" of the volume is not managed here but on the infra side of things, so I don't think we should use the `persistent` word here.
And the `gradle-cache` will likely make readers believe we're talking about the "classic" gradle build cache.

Reply:
Sure